src/TestCase.php

```diff
@@ -198,4 +198,26 @@ protected function getEnvironmentSetUp($app)
     {
         // Define your environment setup.
     }
+
+    /**
+     * Migrate Laravel's default migrations.
+     * 
+     * @param  string $database
+     * 
+     * @return void
+     */
+    public function runLaravelDefaultMigrations($database = null)
+    {
+        $options = ['--path' => '../fixture/migrations'];
```

Comment:
Shouldn't this be just `$options = ['--path' => 'migrations'];`

https://github.com/laravel/framework/blob/5.4/src/Illuminate/Database/Console/Migrations/BaseCommand.php#L20